components/brave_wallet_ui/page/reducers/page_reducer.ts

```diff
@@ -139,10 +144,15 @@ export const createPageReducer = (initialState: PageState) => {
     }
   })
 
-  reducer.on(Actions.setImportWalletError, (state: PageState, payload: ImportWalletError) => {
+  reducer.on(Actions.setImportWalletError, (state: PageState, {
+    hasError,
+    errorMessage,
+    incrementAttempts
+  }: ImportWalletErrorPayloadType) => {
     return {
       ...state,
-      importWalletError: payload
+      importWalletError: { hasError, errorMessage },
+      importWalletAttempts: incrementAttempts ? state.importWalletAttempts + 1 : state.importWalletAttempts
```

Comment:
We can now track the import attempts so that we know when the operation completed